src/outputs/alsa.c

```diff
@@ -1185,6 +1185,99 @@ alsa_write(struct output_buffer *obuf)
     }
 }
 
+static void
+alsa_device_list(const char *hwdev, int card, snd_ctl_card_info_t *info, snd_mixer_selem_id_t *sid)
```

Comment:
Change name to alsa_device_log(), since it is not really listing. And then the other function can be list() like before.